packages/subgraph/src/index.ts

```diff
@@ -474,3 +475,70 @@ ponder.on('RiverAirdrop:Stake', async ({ event, context }) => {
         console.error(`Error processing RiverAirdrop:Stake at blockNumber ${blockNumber}:`, error)
     }
 })
+
+ponder.on('AppRegistry:AppCreated', async ({ event, context }) => {
```

Comment:
We should have another field that tracks whether the app is registered or not, and set it to `false` if AppRegistry:AppUnregistered is emitted. Otherwise, we're going to end up with a db full of zombie apps that have actually been de-registered.

Reply:
good catch, will add isRegistered & isBanned. Will check if there's anything else that we want to add in this table